jekyll/_cci2/contexts.md

```diff
@@ -117,6 +117,17 @@ jobs:
 
 If you move your repository to a new organization, you must also have the context with that unique name set in the new organization.
 
+### Contexts & Environment variables validation
+
+When creating contexts/environment variables, please note the following:
+
+- The context name must be 200 or fewer characters, must contain at least one non-whitespace character, and must not contain leading, trailing or vertical whitespaces.
+- An organization may contain no more than 500 contexts.
+- The environment variable name must be 300 or fewer characters, must have alpha or `_` as the first character and can *only* contain  alphanumeric or `_` as the remaining characters.
+- An environment variable value must be 32000 or fewer characters and must NOT contain `null`. 
```

Comment:
I think we can leave out the bit about `NUL` byte - it's not doc'd by any of our competitors, but it's invalid for any POSIX environment and I don't know of a significant use case for it.
```suggestion
- An environment variable value must have 32K or fewer characters. 
```